src/navigation/HomeNavigation.js

```diff
@@ -20,27 +26,64 @@ const TAB_SCREEN_OPTIONS = {
 	lazy: false //added to ensure that player loads correctly on all tabs
 };
 
-const HomeNavigation = () => (
-	<Tab.Navigator screenOptions={TAB_SCREEN_OPTIONS}>
-		{HOME_SCREENS_ARRAY.map((item, index) => (
-			<Tab.Screen
-				key={index}
-				name={item.name}
-				tabIcon="home"
-				options={{
-					tabBarIcon: ({ color, size }) => (
-						<MaterialCommunityIcons
-							name={item.tabIcon}
-							color={color}
-							size={size}
-						/>
-					)
-				}}
-			>
-				{item.component}
-			</Tab.Screen>
-		))}
-	</Tab.Navigator>
-);
+const HomeNavigation = () => {
+	const [isUserAnonymous, setIsUserAnonymous] = useState();
+
+	const onAuthStateChanged = (user) => {
+		setIsUserAnonymous(user?.isAnonymous);
```

Comment:
if the authentication changes, set the user to the redux store, and persist it, so that we always have access to it